cityofaustin/atd-moped
# Remove migration that does not match main

## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/9724

This should fix our schema spy issue. 
I do not think it would affect the production database, because the removal of a migration file would not affect the migrations that have already been applied (or marked as ignore) in production. 

I am not sure if this would warrant a minor version bump. Production code is changing, but no changes to any functionality or appearance. 

## Testing
**URL to test:** 


**Steps to test:**

check out and run `./hasura-cluster restart` locally. 
Should be able to spin up the hasura cluster

If you check out the production branch, the hasura cluster will not spin up locally because of these migrations. 


---
#### Ship list
